src/files2move2bin/Config.TRACS

```diff
@@ -77,14 +77,14 @@
 Depth = 300.0   # Double in micrometers
 
 # Width of the read-out strips
-Width = 25.0   # Double in micrometers
+Width = 500 #25.0   # Double in micrometers
```

Comment:
> I've been there, I've done that

It was more about testing how this _"comment->reply->modify->commit_"works than anything else 

(The same way that I'm playing with formatting now)